Automattic/wp-calypso
# Nav Unification: Remove MySitesSidebarUnifiedSwitcher

I don't think this is needed after we've made progress on the
unification.

#### Changes proposed in this Pull Request

* Nav Unification: Remove MySitesSidebarUnifiedSwitcher

#### Testing instructions

* Visit a site with the `nav-unification` flag on and ensure the new calypso sidebar still renders and works correctly.
